packages/next/next-server/lib/head.tsx

```diff
@@ -136,6 +153,11 @@ function reduceComponents(
     .concat(defaultHead(props.inAmpMode))
     .filter(unique())
     .reverse()
+    .map((c) => {
+      shouldWarn =
+        shouldWarn || !SAFE_HEAD_CHECKS.map((fn) => fn(c)).includes(true)
```

Comment:
This will be always true if we have more than 1 element in head? 
Suppose we add <meta> and <title> then this would always warn, even though its correct. Did I understand that right?

Reply:
`headElements` is the mounted instances of `<Head>`, whose `props.children` are the actual elements we want to include. So this is just checking if you have more than one `<Head>`. Adding another title wouldn't cause this to warn -- see the test cases 😄